structuretoolkit/build/materialsproject.py

```diff
@@ -0,0 +1,171 @@
+from typing import Union, List
+from ase.atoms import Atoms
+from structuretoolkit.common.pymatgen import pymatgen_to_ase
+
+
+class MPQueryResults:
```

Comment:
This class really only works with the `HasStructure` interface from `pyiron_atomistics`.  Since we don't have that here, it might be better to replace it completely with just lists of dictionaries.